ktor-utils/common/src/io/ktor/util/internal/LockFreeLinkedList.kt

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
just note: `LockFreeLinkedList` is used in `ktor` in 2 places:
  1. in events - there `kotlinx.coroutines.internal.LockFreeLinkedList` is used, not the copy from `ktor`
  2. in JVM part of `WeakTimeoutQueue` - there copy from `ktor` is used

also, current implementation in ktor - copy not of `native-mt` branch, so make no sense for `nix` here
May be it's better to use only one implementation?

Reply:
Both implementations are different and incompatible. The PR is already huge, so let's try solving this problem separately